os/net/ipv6/sicslowpan.c

```diff
@@ -893,15 +892,16 @@ compress_hdr_iphc(linkaddr_t *link_destaddr)
       /* compession compare with link adress (destination) */
 
       iphc1 |= compress_addr_64(SICSLOWPAN_IPHC_DAM_BIT,
-                                &UIP_IP_BUF->destipaddr,
-                                (uip_lladdr_t *)link_destaddr);
+          &UIP_IP_BUF->destipaddr,
+          (const uip_lladdr_t *)packetbuf_addr(PACKETBUF_ADDR_RECEIVER));
```

Comment:
Shouldn't need to cast to const in these places.

Reply:
Was necessary because uip uses a different type ...